src/main/java/org/jenkinsci/test/acceptance/po/CapybaraPortingLayerImpl.java

```diff
@@ -233,6 +254,25 @@ public WebElement getElement(By selector) {
         return all.get(0);
     }
 
+    /**
+     * Works like  {@link #getElement(org.openqa.selenium.By)} but it waits some time for it to exist
+     *
+     * @param selector The selector for the element to get
+     * @param timeoutSec The maximun time to wait for the element to appear in seconds
+     * @return The finded element or null if not found in the specified time
+     */
+    public WebElement waitAndGetElement(By selector, int timeoutSec) {
```

Comment:
Is is needed to introduce this method. Why is client not calling `waitFor(selector, timeoutSec)`? Hiding any exception behind `null` will actually make it harder to understand _why_ the element was not found.